docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 <!-- Thank you for submitting this PR. You are awesome!
 -->
 
-**Status**:
-- [ ] Ready
-- [ ] In Development
-- [ ] On Hold
+**Checklist**
```

Comment:
Do we need this checklist?

1. If the branch is not upto date, that will show up as conflicts. IMO it's a common "git" thing and we don't really need to include it in a checklist.

2. The documentation thing won't be valid every time. Even if it would be, it would be very specific to the change. I don't think we should put this in the PR template. The `template` should be very standard and reflect the needs/goals of all PRs.

Reply:
> If the branch is not upto date, that will show up as conflicts. IMO it's a common "git" thing and we don't really need to include it in a checklist.

@nikhita It's better that as an beginner if they already pulled the latest codebase than their is no conflict. As beginners are not as much as familiar with git so they are afraid of these things.

> The documentation thing won't be valid every time. Even if it would be, it would be very specific to the change. I don't think we should put this in the PR template. The template should be very standard and reflect the needs/goals of all PRs.

Each PR don't need this documentation, if the PR  is quite big so it's need brief documentation so we could save long discussion what i think atleast.

P.S: If you want, I could remove these checklist

Reply:
> It's better that as an beginner if they already pulled the latest codebase than their is no conflict. As beginners are not as much as familiar with git so they are afraid of these things.

Good point. We can probably keep it and see how the contributors react to this/use this. :+1: 

> Each PR don't need this documentation, if the PR is quite big so it's need brief documentation so we could save long discussion what i think atleast.

Sounds good. :+1: